cloudbuild.yaml

```diff
@@ -5,6 +5,7 @@ substitutions:
   '_BASE_IMAGE_FAMILY': 'cos-dev' # base image family
   '_OUTPUT_IMAGE_PREFIX': 'confidential-space'
   '_OUTPUT_IMAGE_SUFFIX': ''
+  '_OUTPUT_IMAGE_FAMILY': ''
```

Comment:
Why do we need image family in the dev project?

Reply:
Need a way for the e2e test to pick the latest dev image, so have that here